### PR TITLE
Add React Boilerplate

### PR DESCRIPTION
**Pull request by Andrewgdewar**

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    default_reviewers:
+      - "karelhala"
```
Comment (josejulio) [inline .dependabot/config.yml:7]: I don't think karelhala would like to review the PRs of this project :D

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import { getRegistry as _getRegistry } from '@redhat-cloud-services/frontend-components-utilities/Registry';
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import pckg from '../package.json';
+import { CustomChromeApi } from './entry';
+import { Routes } from './Routes';
+
+// Example of how to re-implement inaccurately typed imports
```
Comment (josejulio) [inline src/App.tsx:14]: :+1:

File: src/Components/Error/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { ErrorBoundaryPage as _ErrorBoundaryPage } from '@redhat-cloud-services/insights-common-typescript';
```
Comment (josejulio) [inline src/Components/Error/ErrorPage.tsx:1]: I migrated this component to `frontend-components` under the name `ErrorBoundary`

I guess this is the code change needed, but didn't confirm. 
```suggestion
import { ErrorBoundary as _ErrorBoundaryPage } from '@redhat-cloud-services/frontend-components';
```

File: .dependabot/config.yml (hunk 1 above)
Comment (Andrewgdewar) [reply .dependabot/config.yml:7]: OH! awesome thank you. lol

File: src/Components/Error/ErrorPage.tsx (hunk 3 above)
Comment (josejulio) [reply src/Components/Error/ErrorPage.tsx:1]: `3.9.1` (or later if any) surelly has it

Comment (Andrewgdewar) [reply src/Components/Error/ErrorPage.tsx:1]: OH! Correction, updated and found it!

Comment (Andrewgdewar) [reply src/Components/Error/ErrorPage.tsx:1]: The first design seems cleaner than the new one in front-end components. Has design finalized their review of this yet? (Just curious).

Before: 
Screen Shot 2022-06-01 at 1 49 36 PM

After: 
Screen Shot 2022-06-01 at 1 50 20 PM

Comment (Andrewgdewar) [reply src/Components/Error/ErrorPage.tsx:1]: Cool!

**Pull request merged**